docs/user-guide/building-assistants.rst

```diff
@@ -620,12 +675,38 @@ business logic. Read more about this `here <https://rasa.com/docs/rasa/core/form
 However, you should make sure not to handle any unhappy paths inside the form. These
 should be handled by writing regular stories, so your model can learn this behaviour.
 
+
+.. note::
+    Here's a minimal checklist of what you need to do to handle business logic using a form action:
+
+      - ``actions.py``: Define the form action
+              - Define the ``required_slots``, ``slot_mappings`` and ``submit`` methods
+      - ``data/nlu.md``: 
+          - Add examples for an intent to activate the form
+          - Add examples for an ``inform`` intent to fill the form
+      - ``domain.yml``: 
+          - Add all slots required by the form
+          - Add ``utter_ask_{slot}`` responses for all required slots
+          - Add your form action to the ``forms`` section 
+          - Add all intents and entities from your NLU training data
+      - ``data/stories.md``: 
+          - add a story for the form
+      - ``config.yml``: 
+          - Add the ``FormPolicy`` to your policies
+          - Add entity extractors & dimensions (according to which entities will fill your slots) to your pipeline
+      - ``endpoints.yml``: Define the ``action_endpoint``
```

Comment:
same with this as in the previous version

Reply:
did we want checklists like this for all other sections below as well?

Reply:
I'm putting them in - since these sections aren't quite so didactic, it gets a bit wordy. can remove it if its distracting

Reply:
what do you mean? you're putting them into other sections as well?

Reply:
I did in the last commit - they're at the end of the unexpected input/failing gracefully/ML sections

Reply:
ah ok i see!

Reply:
should they stay?